config/rbac/role.yaml

```diff
@@ -40,6 +40,15 @@ rules:
   - patch
   - update
   - watch
+- apiGroups:
+  - apps
+  resources:
+  - deployments
+  - replicasets
+  verbs:
+  - get
+  - list
+  - watch
```

Comment:
please don't forget to add this to the helm chart as well

Reply:
:+1: